dev-python/pysnmp-mibs/pysnmp-mibs-0.1.6.ebuild

```diff
@@ -15,5 +16,4 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
 DEPEND=">=dev-python/pysnmp-4.2.3[${PYTHON_USEDEP}]"
```

Comment:
Not saying you have to do it right now but when bumping to EAPI 7 it's nice to check deps (esp. RDEP vs BDEP). I don't think this ebuild needs any build-time dependencies unless upstream does something horrible.

Reply:
Thanks, indeed, we don't need build-time deps here